src/main/scala-2.11/org/apache/james/gatling/jmap/JmapAuthentication.scala

```diff
@@ -1,25 +1,27 @@
 package org.apache.james.gatling.jmap
 
 
+import java.util.concurrent.TimeUnit
+
 import io.gatling.core.Predef._
 import io.gatling.http.Predef._
 
+import scala.concurrent.duration.Duration
+
 class JmapAuthentication extends Simulation {
 
-  val domain:String = "domain-jmapauthentication.tld"
-  val username:String = "username@" + domain
-  val password:String = "password"
+  val domain = "domain-jmapauthentication.tld"
+  val username = "username@" + domain
+  val password = "password"
 
   val httpProtocol = http
     .baseURL("http://127.0.0.1")
-    .acceptHeader("application/json, text/plain")
+    .acceptHeader("application/json")
+    .contentTypeHeader("application/json; charset=UTF-8")
 
   def obtainContinuationToken(username: String) = exec(
     http("obtainContinuationToken")
       .post("/authentication")
-      .headers(Map(
-        "Content-Type" -> "application/json; charset=UTF-8",
-        "Accept" -> "application/json"))
       .body(StringBody("{\"username\": \"" + username + "\"," +
```

Comment:
you still have a lot of useless escaping and concat because you don't use multiline and substitution